flap.md

```diff
@@ -160,6 +169,7 @@ Flap Semantics
          <flap-live> true </flap-live>
          <flap-ttl> TTL </flap-ttl>
          <flap-beg> BEG </flap-beg>
+         <frame-events> _ => ListItem(FlapTend(ID, LOT, BID)) </frame-events>
```

Comment:
Do we need these log events? As far as I can tell, you can be equally well served by looking for `LogNote(Flap . tend _ _ _)` in your ViolationFSM below.

Reply:
Removed them.